### PR TITLE
Updated map descriptions and landfill tech from maps named in issue #906

### PR DESCRIPTION
**Pull request by Jayefuu**

### CODE REVIEW

File: map_gen/maps/grid_islands_rotated.lua [hunk 1]
```diff
@@ -177,4 +197,14 @@ map = b.any {map, paths, sea}
 map = b.change_map_gen_collision_tile(map, 'water-tile', 'grass-1')
 map = b.rotate(map,degrees(45))
 
+
+-- Disable landfill technology
+local function on_init()
+    local player_force = game.forces.player
+    player_force.technologies['landfill'].enabled = false           -- disable landfill
```
Comment (SimonFlapse) [inline map_gen/maps/grid_islands_rotated.lua:204]: You removed this part:
> -- For best balance run the following commands after map generation:
> -- /silent-command game.forces["player"].technologies["landfill"].enabled = false
> -- /silent-command game.forces.player.character_running_speed_modifier = 1.5
> -- /silent-command game.difficulty_settings.technology_price_multiplier=2

Should we include it in the on_init() function?

```suggestion
    player_force.technologies['landfill'].enabled = false           -- disable landfill
    player_force.character_running_speed_modifier = 1.5
    game.difficulty_settings.technology_price_multiplier = 2
```

Comment (Jayefuu) [reply map_gen/maps/grid_islands_rotated.lua:204]: I think I tested that and it didn't work.  I didn't have time to investigate why.

Comment (SimonFlapse) [reply map_gen/maps/grid_islands_rotated.lua:204]: I just double checked them, and they should work unless it's being set else where

Comment (Jayefuu) [reply map_gen/maps/grid_islands_rotated.lua:204]: No idea what I was doing wrong then. If you've tested the changes then they should be committed.

**Comment by SimonFlapse** [comment]
Does the other grid island also need the technology and running speed increase?

**Review by SimonFlapse — APPROVED** [review]

File: map_gen/maps/gears.lua [hunk 2]
```diff
@@ -14,12 +14,24 @@ RS.set_map_gen_settings(
 
 ScenarioInfo.set_map_name('Gears')
 ScenarioInfo.set_map_description(
-[[
-Am I inside an assembling machine?
-
+    [[
+Large gear shaped islands
+
```
Comment (SimonFlapse) [inline map_gen/maps/gears.lua]: To fix the whitespace
```suggestion
```

**Review by theorangeangle — APPROVED** [review]

**Pull request merged**